### DISCUSSION TITLE
Error Catching

### DISCUSSION THREAD

**Discussion by thejoeker12** [univ-of-utah-marriott-library-apple/python-jamf#51] [Q&A]
Hey! 

Awesome lib, how do I catch an API error? 

Error is jamf.api.APIError(), but I do not know what args to give it? 

Thanks! 
Joseph

**Comment by SamBaRufus** [comment]
I designed `APIError` to so you could do this:

```python
from jamf.api import APIError

try:
    # make your call
    computers = api.get('computers')
except APIError as e:
    # catch APIError here
    pass
```

however, after reviewing the code, it looks like `APIError` now calls `os._exit()`, so if the `try/except` block above doesn't work. I'll have to fix it.
